src/ai/backend/manager/registry.py

```diff
@@ -3046,6 +3046,38 @@ async def commit_session(
                 resp: Mapping[str, Any] = await rpc.call.commit(str(kernel["id"]), email, filename)
         return resp
 
+    async def get_agent_local_config(
+        self,
+        agent_id: AgentId,
+        agent_addr: str,
+    ) -> Mapping[str, str]:
+        async with RPCContext(
+            agent_id,
+            agent_addr,
+            invoke_timeout=None,
+        ) as rpc:
+            return await rpc.call.get_local_config()
+
+    async def get_abusing_report(
+        self,
+        session_name_or_id: Union[str, SessionId],
+        access_key: AccessKey | None = None,
+    ) -> Optional[Mapping[str, str]]:
+        if access_key is not None:
+            kernel = await self.get_session(session_name_or_id, access_key)
```

Comment:
There's no need to get access key information if we already know a reference to session (either name or access key). If you are intending to add user validation logic so that only session owner can retrieve abusing report info, how about setting `access_key` as required variable, removing below logic which resolves session  information without access key, and updating `ComputeContainer.resolve_abusing_report()` to supply access key?

Reply:
ok, I change `access_key` to required parameter and pass the value to the resolver.